### PR TITLE
Format docs according to new guidelines 

### PR DESCRIPTION
**Pull request by bossinc**
https://github.com/grafana/technical-documentation/wiki/Data-source-updates-guideline

https://github.com/grafana/grafana-org/issues/53

### CODE REVIEW

File: docs/sources/_index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+- [Configure the Falcon LogScale data source](/docs/plugins/grafana-falconlogscale-datasourcelatest/configure/)
+- [Falcon LogScale query editor](/docs/plugins/grafana-falconlogscale-datasourcelatest/editor/)
+- [Falcon LogScale templates and variables](/docs/plugins/grafana-falconlogscale-datasourcelatest/templates-and-variables/)
+
+## Requirements
+
+The Falcon LogScale data source has the following requirements:
+
+- An Falcon LogScale instance.
```
Comment (alyssabull) [inline docs/sources/_index.md:26]: nit: A Falcon LogScale instance.

File: docs/sources/_index.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- Any free or paid [Grafana Cloud](https://grafana.com/pricing/) plan or an [activated on-prem Grafana Enterprise license](https://grafana.com/docs/grafana/latest/enterprise/license/activate-license/). Contracted Cloud customers should refer to their agreement.
+
+## Compatibility requirements
+
+There are no compatibility requirements for the Falcon LogScale plugin.
```
Comment (alyssabull) [inline docs/sources/_index.md:31]: nit: should this say data source instead of plugin?

File: docs/sources/_index.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+
+## Get the most out of the Falcon LogScale plugin
```
Comment (alyssabull) [inline docs/sources/_index.md:34]: same as above for consistency?

File: docs/sources/configure.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+Custom HTTP Header Data sources managed by provisioning within Grafana can be configured to add HTTP headers to all requests going to that data source. The header name is configured in the `jsonData` field, and the header value should be configured in secureJsonData. For more information about custom HTTP headers, refer to [Custom HTTP Headers](https://grafana.com/docs/grafana/latest/administration/provisioning/#custom-http-headers-for-data-sources).
+
+## LogScale Token Authentication
+
+You can authenticate using your personal LogScale token. To generate a personal access token, log into LogScale and navigate to User Menu > Manage Account > Personal API Token. Then, set or reset your token. Copy and paste the token into the token field.
+
+# Forward OAuth Identity
+
+**Note: The feature is experimental, which means it may not work as expected, it may cause Grafana to behave in an unexpected way, and breaking changes may be introduced in the future.**
```
Comment (alyssabull) [inline docs/sources/configure.md:44]: nit: "This feature..."

File: docs/sources/editor.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+| [timeChart](https://library.humio.com/falcon-logscale/functions-timechart.html) | Groups data into time buckets. This is useful for time series panels.                | `timeChart(span=1h, function=count())` |
+| [table](https://library.humio.com/falcon-logscale/functions-table.html)         | Returns a table with the provided fields.                                            | `table([statuscode, responsetime])`    |
+| [groupBy](https://library.humio.com/falcon-logscale/functions-groupby.html)     | Group results by field values. This is useful for bar chart, stat, and gauge panels. | `groupBy(_field, function=count())`    |
+
+## Explore view
+
+The Explore view allows you to run LQL queries and visualize the results as logs or charts. For more information about Explore, refer to [Explore](https://grafana.com/docs/grafana/latest/features/explore/). For more information about Logs in Explore, refer to [Explore logs](https://grafana.com/docs/grafana/latest/explore/logs-integration/).
+
+Grafana v9.4.8 and later allows you to create data links in Tempo, Grafana Enterprise Traces, Jaeger, and Zipkin that target Falcon LogScale.
```
Comment (alyssabull) [inline docs/sources/editor.md:34]: Should this be in the data links section?

**Review by alyssabull — APPROVED** [review]
Looks good! Just left a few comments 😄

**Review by aangelisc — APPROVED** [review]

**Pull request merged**